hubblestack/extmods/modules/audit.py

```diff
@@ -109,342 +103,285 @@
 
 log = logging.getLogger(__name__)
 
-hubble_status = HubbleStatus(__name__, 'top', 'audit')
-
-__audit__ = None
+hubble_status = HubbleStatus(__name__, 'top', 'run')
+BASE_DIR_NOVA_PROFILES = 'hubblestack_nova_profiles_v2'
```

Comment:
Please remove all references of 'nova'. Use only audit.

Reply:
Left in config files. We will remove it from config later.